packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -475,6 +475,9 @@ class ExpansionTile extends StatefulWidget {
 
   /// {@macro flutter.material.Material.clipBehavior}
   ///
+  /// If this is not null and a custom collapsed or expanded shape is provided,
```

Comment:
To prevent a common mistake, why not make the default Clip.antiaAlias when a shape is specified? The developer can always override the default of course.

Reply:
Excellent idea! We will no longer to specify shapes as well as a custom `clipBehavior`.  
```diff
          collapsedShape: RoundedRectangleBorder(
            borderRadius: BorderRadius.all(Radius.circular(30.0)),
            side: BorderSide(color: Colors.black, width: 2.0),
          ),
          shape: RoundedRectangleBorder(
            borderRadius: BorderRadius.all(Radius.circular(30.0)),
            side: BorderSide(color: Colors.black, width: 2.0),
          ),
--       clipBehavior: Clip.antiAlias,
```

Just pushed a commit with updated default `clipBehavior`, docs and new tests.